### PR TITLE
Make graphql a peer dependency

### PR DESCRIPTION
**Pull request by jaydenseric**
Fixes https://github.com/graphcool/graphql-config/issues/79.

### CODE REVIEW

**Comment by schickling** [comment]
LGTM @kbrandwijk @asiandrummer ?

**Comment by kbrandwijk** [comment]
I really want to do a lot of testing on this first, also seeing if this works well with how deps are defined on packages using `graphql-import`, because if `graphql` becomes a peer dep, then all of these packages should make sure they install it (like `graphql-cli`). So before merging this, I want to make sure all keeps working downstream.

**Comment by jaydenseric** [comment]
If it's a semver major update there is no problem; it is up to the consumers to install the update and peer dep as needed. Right now the ecosystem is broken so publishing this tweak will fix more problems that it might hypothetically introduce.

**Comment by schickling** [comment]
@kbrandwijk would be great to get an update on this so we can move forward on merging this. :shipit:

File: package.json [hunk 1]
```diff
@@ -55,16 +55,19 @@
     "babel-preset-es2015": "6.24.1",
     "babel-preset-stage-0": "6.24.1",
     "cpx": "1.5.0",
+    "graphql": "^0.12.3",
```
Comment (wtgtybhertgeghgtwtg) [inline package.json]: Wanna bump this to `^0.13.0` and increase the peerDependency version range?

**Pull request merged**